backend/movieh/settings.py

```diff
@@ -11,6 +11,7 @@
 """
 
 from pathlib import Path
+import os
 
```

Comment:
Use https://pypi.org/project/dj-database-url/ for database connection configuration
```suggestion
import dj_database_url
```

Reply:
Also install that package and add to `requirements.txt`